charts/prometheus/templates/_helpers.yaml

```diff
@@ -83,3 +83,46 @@ imagePullSecrets:
 {{- .Values.securityContext | toYaml | nindent 12 }}
 {{- end -}}
 {{- end }}
+
+{{- define "prometheus.airflowMetricsActionDrop" }}
+- source_labels: [instance]
+  regex: "([-a-z0-9]*[a-z0-9])-.*?"
+  replacement: "$1"
+  target_label: deployment
```

Comment:
This is not an `action: drop`. Does it belong in this `prometheus.airflowMetricsActionDrop` block?